pkgs/development/python-modules/narwhals/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  hatchling,
+}:
+let
+  pname = "narwhals";
```

Comment:
`narwhals` is in already in `staging` (https://github.com/NixOS/nixpkgs/pull/345326/commits/6432b6e7a26edcde8f50d62fc5d4c8e4bde77af0).

Reply:
This PR now conflicts with master